Jenkinsfile

```diff
@@ -73,9 +73,14 @@ pipeline {
           withBeatsEnv(archive: false, id: "lint") {
             dumpVariables()
             setEnvVar('VERSION', sh(label: 'Get beat version', script: 'make get-version', returnStdout: true)?.trim())
-            cmd(label: "make check-python", script: "make check-python")
-            cmd(label: "make check-go", script: "make check-go")
-            cmd(label: "Check for changes", script: "make check-no-changes")
+            whenTrue(env.ONLY_DOCS == 'true') {
+              cmd(label: "make check", script: "make check")
+            }
+            whenTrue(env.ONLY_DOCS == 'false') {
+              cmd(label: "make check-python", script: "make check-python")
+              cmd(label: "make check-go", script: "make check-go")
```

Comment:
Will these checks check the docs? Should we run `make check` also here?

Reply:
I'm a bit confused, IIUC the issue was when PRs are related for DOCS only changes since the `linting` does not run `make check`.

`make check` was refactored to run a basic linting and then run the linting specifically to the beats that should be validated and this help to reduce the linting time from a ~15mnts to ~5 minutes.

Then, I implemented the below:
* If only docs changes then run `make check`
* If changes contain more than just docs then It runs as usual.

Reply:
>     * If changes contain more than just docs then It runs as usual, since the beats specific stages are the ones in charge for running their linting.

Oh ok, all right :+1: